### PR TITLE
Add two new rules.

### PR DESCRIPTION
**Pull request by ralphbean**
One for watching packages with the acl commit and
another for watching packages with the watchcommits flag.

Fixes fedora-infra/fmn#93.

### CODE REVIEW

File: fmn/rules/utils.py [hunk 1]
```diff
@@ -148,10 +148,7 @@ def _get_pkgdb2_packages_for(config, username):
 
     data = req.json()
 
-    packages_of_interest = \
-        data['point of contact'] + \
-        data['co-maintained'] + \
-        data['watch']
+    packages_of_interest = sum([data[flag] for flag in flags], [])
```
Comment (pypingou) [inline fmn/rules/utils.py]: sum()?

Comment (ralphbean) [inline fmn/rules/utils.py]: Yeah!

As its second argument, ``sum()`` takes the object that it should *start* the sum with:

```python
>>> a, b, c = list(range(2)), list(range(3)), list(range(1))
>>> sum([a, b, c], [])
[0, 1, 0, 1, 2, 0]
>>> sum([1, 1, 1], 0)
3
>>> sum([1, 1, 1], 20)
23
```

Comment (pypingou) [inline fmn/rules/utils.py]: Ouh, ``sum()`` works on list as well, cool :)

**Comment by pypingou** [comment]
Sorry, took me a little while to figure the code out, :+1: for me

**Comment by ralphbean** [comment]
Thanks @pypingou!

**Pull request merged**